plugins/idea/src/main/java/org/robovm/idea/running/RoboVmIOSRunConfigurationSettingsEditor.java

```diff
@@ -354,6 +420,20 @@ private void updatePairedWatch(boolean moduleChanged, Boolean valueToSet) {
             Decorator<Module> moduleSelected = Decorator.from(module);
             Config config = moduleSelected != null ? RoboVmPlugin.loadRawModuleConfig(moduleSelected.data) : null;
             moduleHasWatchApp = config != null && config.getWatchKitApp() != null;
+
+
+
+            if (config != null) {
+                Properties properties = config.getProperties();
+                InfoPList infoPList = config.getInfoPList();
+                infoPList.parse(properties);
```

Comment:
there is going to be NPE or next line should check `infoPList.getBundleIdentifier()` for null